src/iphymv.cpp

```diff
@@ -135,6 +135,10 @@ usage() {
         "  iphymv -S demoResc -R 'pt1;r1;s1' /tempZone/home/alice/test.txt",
         "  iphymv -S demoResc -R 's1' /tempZone/home/alice/test.txt",
         " ",
+        "If the leaf resource targeted for a phymv refers to the archive of a compound",
+        "resource hieararchy, the DIRECT_ARCHIVE_ACCESS will be returned because",
```

Comment:
hieararchy -> hierarchy

Reply:
and either remove 'the' before DIRECT_ARCHIVE_ACCESS or add the word 'error' after it

Reply:
Done